runtime/doc/treesitter.txt

```diff
@@ -1386,7 +1386,7 @@ parse({lang}, {query})                          *vim.treesitter.query.parse()*
       • |vim.treesitter.query.get()|
 
                                                        *Query:iter_captures()*
-Query:iter_captures({node}, {source}, {start}, {stop})
+Query:iter_captures({node}, {source}, {start}, {stop}, {opts})
```

Comment:
```suggestion
Query:iter_captures({node}, {source}, {start_row}, {stop_row}, {opts})
```

May be worth changing `stop` -> `end` since that's what we use everywhere else.